src/test/java/org/folio/rest/api/AsyncMigrationTest.java

```diff
@@ -93,6 +94,7 @@ public void canGetAvailableMigrations() {
   }
 
   @Test
+  @Ignore
```

Comment:
An `@Ignore` should have a reason: `@Ignore("Succeeds on Jenkins but fails consistently on local machines")`

Reply:
This test succeeds locally and on Jenkins:
https://jenkins-aws.indexdata.com/job/folio-org/job/mod-inventory-storage/job/FOLIO-3388-with-canCancelMigration/
https://github.com/folio-org/mod-inventory-storage/tree/FOLIO-3388-with-canCancelMigration

Please explain why the `@Ignore` is needed.

Reply:
Thanks for noting.